tycho/template/pod.yaml

```diff
@@ -68,6 +68,21 @@ spec:
     volumeMounts:
     - name: {{ system.stdnfs_pvc }}
       mountPath: {{ system.parent_dir }}
+  {% if system.irods_enabled == True %}
+  - name: nfsrods-prep
+    image: busybox:1.28
+    # Fixed resources
+    command: ['sh', '-c'] 
+    args:
+    - touch /permissions/irodsvalidusers &&
+      echo {{system.nfsrods_uid}} >> /permissions/irodsvalidusers
```

Comment:
This might get big after a while.  Maybe check to see if the username is already in there and if it is then don't add it?

Reply:
Does this file go away when the pod is terminated?  If not, then will it end up having multiple entries for a single username if they start multiple apps?

Reply:
Yes, it is in ephemeral storage tied to the pod so it disappears when the pod dies. So with multiple apps started, each pod will get its own permissions file in its own separate PVC. Now, I guess what this brings up that I didn't think about, is how a deployment of NFSRODS will track the creation of these new PVC's.
<img width="1285" alt="image" src="https://user-images.githubusercontent.com/104910179/205092674-c711d671-fd48-4cf2-9b83-9016e5942998.png">